pkg/core/blockchain.go

```diff
@@ -131,7 +140,7 @@ func (bc *Blockchain) init() error {
 	// At this point there was no version found in the storage which
 	// implies a creating fresh storage with the version specified
 	// and the genesis block as first block.
-	log.Infof("restoring blockchain with version: %s", version)
+	bc.log.Info("restoring blockchain with version", zap.String("version", version))
```

Comment:
I think `restoring blockchain` will be enough here, given that we log `version` separately now.